app/_data/docs_nav_gateway_3.5.x.yml

```
@@ -217,14 +217,16 @@ items:
         url: /production/canary
       - text: Clustering Reference
         url: /production/clustering
-      - text: Logging
+      - text: Logging and Debugging
         items:
           - text: Log Reference
             url: /production/logging/log-reference
           - text: Dynamic log level updates
             url: /production/logging/update-log-level-dynamically
           - text: Customize Gateway Logs
             url: /production/logging/customize-gateway-logs
+          - text: Debug Requests
+            url: /production/logging/debug-request
```

```suggestion
            url: /production/debug-request
```
I'd do this, since debugging isn't logging, even though the two are related.